src/pages/category-new/category-page/category-page.js

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { push } from 'connected-react-router';
+import { Link } from 'react-router-dom';
+import { Button, Typography } from '@material-ui/core';
+import { useStyles } from './category-page.styles';
+import {
+  getCategories,
+  deleteCategory
+} from '../../../redux/category-new/category.actions';
+import { closeDialog } from '../../../redux/dialog-window/dialog-window.actions';
+import useSuccessSnackbar from '../../../utils/use-success-snackbar';
+import TableContainerRow from '../../../containers/table-container-row';
+import TableContainerGenerator from '../../../containers/table-container-generator';
+import LoadingBar from '../../../components/loading-bar';
+import { config } from '../../../configs';
+
+const { CATEGORY_REMOVE_MESSAGE } = config.messages;
+const { CREATE_CATEGORY, DELETE_CATEGORY } = config.buttonTitles;
+
+const pathToCategoryAddPage = config.routes.pathToAddCategory;
+const tableTitles = config.tableHeadRowTitles.Categories;
+
+const CategoryPage = () => {
+  const styles = useStyles();
+  const { openSuccessSnackbar } = useSuccessSnackbar();
+  const { list, loading } = useSelector(({ Categories }) => ({
+    list: Categories.list,
+    loading: Categories.newsLoading,
```

Comment:
Why it's called newsLoading if we use it for categories.
Please name it correctly

Reply:
I used it becouse it is used like this in anothet our components like model page,pattern page and others.

Reply:
I guess it's still will work with correct naming as @Iryna-Bzdyr said, try it

Reply:
Allready renamed it corectly